src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt

```diff
@@ -338,6 +345,7 @@ class TaskDeploymentService {
                                 "--connectivity-check-url=https://localhost:8000",
                                 "--window-size=${screenSize.toList().joinToString(",")}").joinToString(" "),
                         "VNC_SCREEN_SIZE" to screenSize.toList().joinToString("x"),
+                        "SFTP_PUBLIC_KEY" to sshKeyPair.public,
```

Comment:
Why can we pass the public key as an env variable here? But have to Base64 encode the private key?

Reply:
The public key doesn't have newline characters, the private one does